bitnami/mongodb/values.schema.json

```diff
@@ -1,232 +1,3525 @@
 {
```

Comment:
Could you please keep the `values.schema.json` as it is?

Reply:
I have reverted the changes in the schema